.travis.yml

```diff
@@ -1,22 +1,52 @@
 language: cpp
+
+os:
+  - linux
+  - osx
+
+compiler:
+  - gcc
+  - clang
+
+branches:
+  only:
+    - master
+    - travis-osx
+
+before_script:
+  - if [ "$TRAVIS_OS_NAME" == "osx" ]; then brew update; fi
+  - if [ "$TRAVIS_OS_NAME" == "osx" ]; then brew install git qt4; fi
```

Comment:
I must say I'm confused by the distinction between `before_script`, `install`, and `before_install`.  I'm quite new to Travis and the docs did not really enlighten me.  Regardless, shouldn't both the linux and OSX package manager commands go in the same section?